File: VB.NET/message.vb
'Use in Visual Studio or download mono-basic to compile.
'Takes string input from user and writes to console
Imports System

Public Module Message
    Public Sub Main()
        Dim myText As String
        Console.WriteLine("Enter some text:")
        myText = Console.ReadLine()
        Console.WriteLine("Here is your text:" & vbCrLf & $"{myText}")

    End Sub
End Module

File: Python/message.py
#!/usr/bin/python
# Use with either python3 message.py or chmod +x and ./message.py (python will refer to python 2)
# Takes string input from user and writes to console.

def main():
    myText = input("Enter some text:\n")
    print (f'Here is your text:\n{myText}')

if __name__ == "__main__":
    main()
